modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -90,6 +90,10 @@ the `..*` descendants selector on a key to select values from a flattened list.
 | Defines a function that recursively checks an element and all of its children for a specific XML attribute and removes it with `-`. It also uses `mapObject`, `if` and `is` (to match types).
 
 
+| xref:dataweave-cookbook-pass-xml-attributes.adoc[Pass XML Attributes]
+| Pass XML attributes from the input source payload to the output XML. Use the `map` function to iterate over the XML attributes items.
```

Comment:
```suggestion
| Pass XML attributes from the input source payload to the output XML. It uses the attribute expression plus a dynamic expression `@((dynamicExpression))` to create the attributes of the output tag from the input.
```
Passing the xml attributes is because the `@((dynamic attributes))`. The map is just iterating over the array of NameDetails.